shell/src/main/java/alluxio/shell/command/LeaderCommand.java

```diff
@@ -44,14 +46,17 @@ public int getNumOfArgs() {
 
   @Override
   public int run(CommandLine cl) {
-    FileSystemMasterClient client = FileSystemContext.INSTANCE.acquireMasterClient();
+    FileSystemMasterClient client = null;
```

Comment:
To get this simpler, can you just use `try-with-resource` pattern like
```java
try (CloseableResource<FileSystemMasterClient> client = FileSystemContext.INSTANCE.acquireMasterClientResource()) {
  ...
}
```
instead of `try-catch-finally`